From 90c76fbd6069b8e1e98236f31317ed22792ab739 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <luca.boccassi@gmail.com>
Date: Fri, 28 Apr 2017 16:08:46 +0100
Subject: [PATCH] Problem: new zmq_poller used by zmq_poll without DRAFTs

Solution: do not define ZMQ_HAVE_POLLER in src/zmq_drafts.h otherwise
src/zmq.cpp will implement zmq_poll using the new poller classes.
Same for ZMQ_HAVE_TIMERS, even though it has no internal effect, but
to be safe against future development.
---
 src/zmq_draft.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/zmq_draft.h b/src/zmq_draft.h
index bfbf9e3..9aed6dd 100644
--- a/src/zmq_draft.h
+++ b/src/zmq_draft.h
@@ -67,8 +67,6 @@ const char *zmq_msg_group(zmq_msg_t *msg);
 /*  Poller polling on sockets,fd and thread-safe sockets                      */
 /******************************************************************************/
 
-#define ZMQ_HAVE_POLLER
-
 typedef struct zmq_poller_event_t
 {
     void *socket;
@@ -103,8 +101,6 @@ int zmq_poller_remove_fd (void *poller, int fd);
 /*  Scheduling timers                                                         */
 /******************************************************************************/
 
-#define ZMQ_HAVE_TIMERS
-
 typedef void (zmq_timer_fn)(int timer_id, void *arg);
 
 void *zmq_timers_new (void);
